src/arguments.cpp

```diff
@@ -182,9 +182,21 @@ std::tuple<IpVer, std::string, uint8_t> Arguments::asIpAddrMask()
             }
         }
     }
+    else
+    {
+        const std::optional<IpVer> ver = isIpAddress(arg);
+        if (ver.has_value())
+        {
+            constexpr uint8_t ip4Prefix = 24;
```

Comment:
Why not 16? Or 8?
I would define it depending on the IP address (10.x.x.x => 8, 192.168.x.x => 16, etc).
But anyway, I think it is a bad idea to guess the mask. User should specifiy it manually for static addresses.

Reply:
I think that 24 is the most popular mask for IPv4.  Nobody uses the big network like `10.x.x.x/8` without the splitting on subnets.
But I hope in the most cases the mask will be specified.

Reply:
I agree, it is common practice to assume /24 as default mask for IPv4